lib/client/hot.js

```diff
@@ -79,12 +84,16 @@ function check() {
       if (upToDate()) {
         log.info('App is up to date.');
       }
-    });
+    }
+
+    if (options.reload) next(modules);
```

Comment:
I'm not sure why the linter didn't catch this (I'll look into that) but please don't use single line if/else blocks. 

```
if (...) {
} else {
}
```
please. same for line 96.